tutorials/NWB Tutorials/read_in_NWB.m

```diff
@@ -0,0 +1,48 @@
+%% *MatNWB Setup*
+% Start by setting up your MATLAB workspace. The code below adds the directory 
+% containing the MatNWB package to the MATLAB search path. MatNWB works by automatically 
+% creating API classes based on a defined schema. Running the generateCore() function
+% generates these classes for the lastest schema version.
+
+path_to_matnwb = '~/Repositories/matnwb';% change to your own path location
+addpath(genpath(pwd));
+generateCore();
+%% *Load in Image Files*
+% The extractor.m function function requires two inputs
+
+%  1) An image stack with dimensions [rows, columns, frames]
+%  2) A config strcuture with the parameters for the EXTRACT algorithm
+
+
+% You can read in an nwb file with the nwbRead function from MatNWB. The code 
+% below reads in a file downloaded from this DANDIset <https://gui.dandiarchive.org/#/dandiset/000054 
+%For this file, the raw images are located as a DataStub within 
+% the TwoPhotonSeries object. In MatNWB, the data is read "lazily". This means 
+% that, instead of reading the entire dataset into memory, you have the option 
+% of reading just a subset of the data stored on disk. The code below reads into 
+% memory only the first 2,000 frames of the data. You can refer to the MatNWB 
+% ophys tutorial to learn more about the structure and contents of NWB files. 
+
+% Load File (change to your own file location)
+inputFilePath = '/Users/cesar/Documents/DANDI_files/sub-F1_ses-20190407T210000_behavior+ophys.nwb';
+nwb = nwbRead(inputFilePath);
+% Load in image stack
+image_stack = nwb.acquisition.get('TwoPhotonSeries').data(:, :, 1:2000);
```

Comment:
@bendichter This line keeps generating the following error: 
```
Index in position 3 exceeds array bounds. Index must not exceed 1.
```

I think it's due to the fact that the nwb file was generated with an older version of the schema. It might be better to revert back to the .load([start],[end]) format. Though not as pretty, it's more robust.

Reply:
Just double-checked and this way of loading in the data doesn't work either with a test file created with the latest schema. Is the format of this line correct? Let me know if there's another MATLAB function/script that makes use of this that I can use for reference.

Reply:
Looked into the for the correct method to do this. For reference, the simplest form of this line is the following:
```matlab
nwb.acquisition.get('TwoPhotonSeries').data.internal.stub(:, :, 1:2000);
```
One has to reference the stub field itself rather than the `DataPipe` container. The `DataStub` object is the one that has the `.load_mat_style()` method which allows for this kind of slicing